ip_load/axis_converter_V1_0_axis_converter.json

```diff
@@ -0,0 +1,11 @@
+{
+   "core_in_width": 128,
+   "core_out_width": 64,
+   "core_user_width": 0,
+   "core_reverse": 0,
+   "build_dir": "./ip_load/ip_load.IPs/RapidSilicon/IP/axis_converter/V1_1/axis_converter/src",
```

Comment:
This is the dummy json generated correct ? The build dir is something we get from ip gen ?

Reply:
Yeah this is dummy data until litex dependencies are fixed and we can switch to testing with real IPs. The build dir is set when the python script prints out the generated json, I edited the build_dir path to make it relative instead of hardcoded to my directories.

Reply:
Thank you.